open-api/rest-catalog-open-api.py

```diff
@@ -502,6 +502,9 @@ class PlanStatus(BaseModel):
 class RegisterTableRequest(BaseModel):
     name: str
     metadata_location: str = Field(..., alias='metadata-location')
+    overwriteRequested: Optional[bool] = Field(
```

Comment:
should use underscore and hyphen for consistency. On the other hand, I think simple `overwrite` is probably enougb

Reply:
thank you Steven and simplified to `overwrite`. Originally I was referring to [purgeRequested in drop table](https://github.com/apache/iceberg/blob/main/open-api/rest-catalog-open-api.yaml#L1113) as an example, but I overlook the existing convention